test/karma.config.js

```diff
@@ -0,0 +1,39 @@
+const { DEBUG } = process.env || 'false';
+
+const files = [
+    '../node_modules/@mparticle/web-sdk/dist/mparticle.js',
+    '../node_modules/mocha/mocha.js',
+    '../node_modules/should/should.js',
+    'config.js',
+    '../dist/CriteoEventForwarder.iife.js',
+    'lib/mockhttprequest.js',
```

Comment:
I see you're referencing `lib/` but that's not where `mockhttprequest.js` lives in this repo. Please move the js file into lib rather than changing this reference so we can make the repo more consistent.